fs/fs-fuzz.py

```diff
@@ -38,7 +38,7 @@ def verify_dmesg(self):
         pattern = ['WARNING: CPU:', 'Oops',
                    'Segfault', 'soft lockup', 'Unable to handle']
         for fail_pattern in pattern:
-            if fail_pattern in self.whiteboard:
+            if fail_pattern in self.whiteboard.decode():
```

Comment:
@harish-24 yes i know it require library patch as well as you we are in process of send that as well , but test supporting python3 still  be ok

Reply:
@PraveenPenguin It does not require library because `genio` expects string as input. So test-case has to change. There is no need for a library patch.